app/src/main/res/layout/fragment_maps.xml

```diff
@@ -47,7 +49,7 @@
         android:layout_width="0dp"
         android:layout_height="0dp"
         android:layout_marginTop="16dp"
-        android:apiKey="AIzaSyAPWYWAau9qrdWdMc5meFMUqrkla2ccPp0"
+        android:apiKey="@string/MAPS_API_KEY"
```

Comment:
@SergeiMikhailovskii I still see keys for facebook and twitter in your `strings.xml` file. Could you please use the same approach as here for these keys?